examples/passing/RecordUnionDupe.purs

```diff
@@ -0,0 +1,12 @@
+module Main where
+
+import Prelude
+import Data.Record (merge)
+import Control.Monad.Eff.Console (log, CONSOLE)
+import Control.Monad.Eff (Eff)
+import Unsafe.Coerce (unsafeCoerce)
+
+main :: Eff (console::CONSOLE) Unit
+main = do
+  let result = (merge {left:1} {left:"Done"}).left
+  log $ unsafeCoerce result
```

Comment:
When I say right biased i mean:

```purescript
RowUnion (same::L1, same::L2) (same::R1, same::R2) -> (same::R1,same::R2,same::L1,same::L2)
```
This is ok still isn't it? 

However, I just realised to get the desired partial application behaviour `merge` could just be changed to:
`merge :: RowUnion b a c => Record a -> Record b -> Record c`

So it's probably best to do it left biased as it's more intuitive to have: 

```purescript
RowUnion (same::L1, same::L2) (same::R1, same::R2) -> (same::L1,same::L2,same::R1,same::R2)
```